src/sql/plot.py

```diff
@@ -561,7 +559,7 @@ def _histogram(table, column, bins, with_=None, conn=None, facet=None):
     else:
         template_ = """
         select
-            "{{column}}" as col, count ({{column}})
+            "{{column}}" as col, count ("{{column}}")
```

Comment:
can we move the uppercasing logic here? e.g., you can check whether the connection is snowflake, and if so uppercase here, this way the API for the user remains the same

Reply:
Added [to_upper_if_snowflake_conn](https://github.com/ploomber/jupysql/pull/697/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R423) to transform the `table` and `column`